ui/src/containers/Header/index.tsx

```diff
@@ -82,6 +133,77 @@ const Header: React.FC = observer(() => {
           </TextContent>
         </Grid>
       </Modal>
+
+      <Modal
+        title="Instructions to add a new Resource in Hub"
+        isOpen={isResInstModallOpen}
+        onClose={() => setResInstModalOpen(false)}
+        width={'50%'}
+      >
+        <Grid>
+          <TextContent>
+            <Text component={TextVariants.h3}>
+              Add a new Resource through Catalogs and follow the below steps
+            </Text>
+            <Text>
+              Add a new resource to Hub by creating a pull request to available Catalogs with the
```

Comment:
```suggestion
              Add a new resource to Hub by creating a pull request to available Catalogs with the
              new resource which must follow the guildelines mentioned [here]() 
```

Reply:
maybe a better wording here
```suggestion
              Add a new resource to Hub by creating a pull request to available Catalogs where the new resource should follow the guildelines mentioned here 

```